st2common/st2common/util/greenpooldispatch.py

```diff
@@ -45,7 +73,23 @@ def _flush(self):
 
     def _flush_now(self):
         if self._dispatcher_pool.free() <= 0:
+            now = time.time()
+
+            if (now - self._pool_last_free_ts) >= POOL_BUSY_THRESHOLD_SECONDS:
+                LOG.info(POOL_BUSY_LOG_MESSAGE % (self.name, POOL_BUSY_THRESHOLD_SECONDS))
```

Comment:
I was thinking about it, but I decided to use INFO.

The thing is that this could represent a problem (warning), but this message being logged doesn't always represent an actual problem so I decided to use INFO.